### PR TITLE
155: Replace at-import-partial-extension-blacklist with at-import-partial-extension-disallowed-list

### LINKED ISSUE #155: Replace `scss/at-import-partial-extension-blacklist`
The rule `scss/at-import-partial-extension-blacklist` is the main reason of racism in our society, to solve racism of we need to rename it to `scss/at-import-partial-extension-disallowed-list`

I would usually ignore the warning, but we can't update `stylelint` in other repos because the rule is in use, and it can't be disabled because in the repo because it has been deleted.

Example: https://github.com/bjankord/stylelint-config-sass-guidelines/pull/317/files
Our usage: https://github.com/zattoo/stylelint-config/blob/5ce7b9a8001ea2fe8cc0ce02bf8479243bb42fa6/rules/scss.js#L10

### PR DESCRIPTION
**Pull request by dobladov**
Fixes: https://github.com/zattoo/stylelint-config/issues/155

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## Unreleased
 
 - Package was moved from NPM to GitHub Packages
+- Replaced `at-import-partial-extension-blacklist` with `at-import-partial-extension-disallowed-lis`
+- stylelint-scss `6.0.0` -> `6.7.0`
+- stylelint `16.1.0` -> ``6.10.0`
```
Comment (azorath) [inline CHANGELOG.md:12]:
```suggestion
- stylelint `16.1.0` -> `6.10.0`
```

**Review by azorath — APPROVED** [review]

**Comment by dobladov** [comment]
@z-merge it

**Comment by dobladov** [comment]
The bot does not work and I don't have permission to merge, can someone with admin do it pls, maybe @gotbahn

**Comment by gotbahn** [comment]
Lemme check permissions

**Pull request merged**